bip-update-process.md

```diff
@@ -0,0 +1,645 @@
+list](https://groups.google.com/g/bitcoindev). This gives the author a chance to collect initial feedback and address
+fundamental concerns. If the author wishes to work in public on the draft at this stage, it is recommended that they
+open a pull request against their personal fork of the BIPs repository instead of the main BIPs repository.
+
+The following sections refer to BIP Status Field values. The BIP Status Field is defined in the Header Preamble
+specification above.
+
+#### Preliminary
+
+After fleshing out the draft further and ensuring that it is of high quality and properly formatted, the author should
+open a pull request to the [BIPs repository](https://github.com/bitcoin/bips). The draft BIP must adhere to the
+formatting requirements specified below and named with a working title of the form "bip-title". The author must not
+self-assign a number to the BIP draft.
+
+BIPs that (1) adhere to the formatting requirements, (2) are on-topic, and (3) have materially progressed
+beyond the ideation phase, e.g. by generating substantial public discussion and commentary from diverse contributors, by
+independent Bitcoin projects working on adopting the proposal, or by the author working for an extended period of time
+towards improving the proposal based on community feedback, will be assigned a number by a BIP editor. The BIP editors
+should delay number assignment when they perceive a BIP draft being met with comprehensive lack of interest: number
+assignment facilitates the distributed discussion of ideas, but before a BIP garners some interest of the Bitcoin
+community, there is no need to refer to it by a number.
+
+BIP drafts are also not eligible for number assignment if they duplicate efforts, disregard formatting rules, are too
+unfocused or too broad, fail to provide proper motivation, fail to address backwards compatibility, or fail to specify
+the feature clearly and completely. Reviewers and BIP editors should provide guidance on how the draft may
+be improved to progress towards readiness. Pull requests that are proposing off-topic or unserious ideas, or have
+stopped to make progress may be closed.
+
+When the BIP draft is complete, a BIP editor will assign a BIP type, and merge it to the BIPs repository.
+After the BIP is merged to the repository, its main focus should no longer shift significantly, even while the author
+may continue to update the draft as necessary in the git repository. Updates to drafts by the author should also be
+submitted as pull requests.
+
+#### Ready[^ready]
+
+When the author is confident that their BIP represents a net improvement, is clear, comprehensive, has a working
+reference implementation (if applicable), and is ready for adoption by the Bitcoin community, they may update the BIP’s
+status to Ready to indicate that they recommend adoption or implementation of the BIP. Where applicable, the
```

Comment:
```suggestion
status to _Ready_ to indicate that they recommend adoption or implementation of the BIP. Where applicable, the
```
Accepting this suggestion may require similar changes elsewhere to be consistent. I'd prefer that any mention of these status values be italicized, but that may be too much change.

Reply:
I had that in an earlier version, but it was too much visual noise for my taste. I did remove all other uses of the status field values (i.e. every appearance of Ready, Abandoned, Active, and Preliminary refers to the status field value).